WordPress/src/main/java/org/wordpress/android/ui/media/MediaUtils.java

```diff
@@ -99,6 +107,65 @@ private static boolean isVideo(String url) {
         return false;
     }
 
+    public static Cursor getWordPressMediaImages() {
+        Blog blog = WordPress.getCurrentBlog();
+
+        if (blog == null) {
+            return null;
+        }
+
+        return WordPress.wpDB.getMediaImagesForBlog(String.valueOf(blog.getLocalTableBlogId()));
+    }
+
+    public static Cursor getWordPressMediaVideos() {
+        Blog blog = WordPress.getCurrentBlog();
```

Comment:
I think we're trying to kill the concept of a 'current blog' in the application class. Would it be possible to pass the blogId along to this method?